List/LocationList.py

```diff
@@ -213,15 +213,15 @@ def getTreasureList():
             KH2Treasure(58, "Santa's House AP Boost", LocationTypes=[locationType.HT]),
             KH2Treasure(301, "Oogie Boogie Magnet Element", LocationTypes=[locationType.HT], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]),
-            KH2Treasure(297, "Present", LocationTypes=[locationType.HT], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]),
-            KH2Treasure(298, "Decoy Presents", LocationTypes=[locationType.HT], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]),
-            KH2Treasure(275, "Decisive Pumpkin", LocationTypes=[locationType.HT], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]),
-            KH2Treasure(544, "Vexen (AS) Road to Discovery", LocationTypes=[locationType.HT, locationType.AS], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]),
-            KH2Treasure(549, "Vexen (Data) Lost Illusion", LocationTypes=[locationType.HT, locationType.AS], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]),
-
-            KH2Treasure(70, "Rampart Naval Map", LocationTypes=[locationType.PR]),
-            KH2Treasure(219, "Rampart Mythril Stone", LocationTypes=[locationType.PR]),
-            KH2Treasure(220, "Rampart Dark Shard", LocationTypes=[locationType.PR]),
+            KH2Treasure(297, "Present", LocationTypes=[locationType.HT], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]).setLocationWeight(10),
+            KH2Treasure(298, "Decoy Presents", LocationTypes=[locationType.HT], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]).setLocationWeight(10),
+            KH2Treasure(275, "Decisive Pumpkin", LocationTypes=[locationType.HT], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]).setLocationWeight(10),
+            KH2Treasure(544, "Vexen (AS) Road to Discovery", LocationTypes=[locationType.HT, locationType.AS], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]).setLocationWeight(10),
+            KH2Treasure(549, "Vexen (Data) Lost Illusion", LocationTypes=[locationType.HT, locationType.AS], InvalidChecks=[itemType.GROWTH_ABILITY, itemType.SUPPORT_ABILITY, itemType.ACTION_ABILITY]).setLocationWeight(10),
+
+            KH2Treasure(70, "Rampart Naval Map", LocationTypes=[locationType.PR]).setLocationWeight(-10),
+            KH2Treasure(219, "Rampart Mythril Stone", LocationTypes=[locationType.PR]).setLocationWeight(-10),
+            KH2Treasure(220, "Rampart Dark Shard", LocationTypes=[locationType.PR]).setLocationWeight(-10),
             KH2Treasure(71, "Town Dark Stone", LocationTypes=[locationType.PR]),
```

Comment:
I think town and cave mouth chests should have an early game weight, although I could be convinced not they are kind of later early game since the town fight can be tough with low tools

Reply:
I tried to avoid putting things past fights in the easy pool. But I think this is a candidate for if item distribution tests show unideal results, we can add more easy spots here.

Reply:
fair enough